compile.sh

```diff
@@ -556,6 +566,14 @@ buildtool="" #Use ninja by default, fall back to make if that is not available.
                -DUSE_XDG=$USE_XDG                              \
                -DUSE_FLTO_IF_AVAILABLE=${USE_FLTO}
 
+    if [ $CORES -gt $MAXCORES ] && [ "$BUILD_TRANSLATIONS" = "ON" ] ; then
```

Comment:
```suggestion
    if [ $CORES -gt $MAXCORES -a "$BUILD_TRANSLATIONS" = "ON" ] ; then
```

Sorry, I missed this previously.